src/isIsogram.test.js

```diff
@@ -6,4 +6,28 @@ describe('isIsogram', () => {
   it(`should be declared`, () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
+  it(`should return true for an empty string`, () => {
+    const res = isIsogram('');
+
+    expect(res).toBe(true);
+  });
+
+  it(`should return true if word has no repeating letters`, () => {
+    const res = isIsogram('playgrounds');
+
+    expect(res).toBe(true);
+  });
+
+  it(`should return false if word has repeating letters`, () => {
+    const res = isIsogram('look');
+
+    expect(res).toBe(false);
+  });
+
+  it(`should be case sensitive`, () => {
```

Comment:
Test description should be 'should be case insensitive' instead of 'should be case sensitive'